tests/utils/test_usage_stats.py

```diff
@@ -30,3 +36,56 @@ class ExampleBentoService(bentoml.BentoService):
     assert properties["artifact_types"]
     assert 'NO_ARTIFACT' in properties["artifact_types"]
     assert properties["env"] is not None
+
+
+def mock_track_cli(event, properties):
+    print(event)
```

Comment:
remove print

Reply:
We need to print out this event for assert later on.